mail-client/trojita/trojita-0.7.0_git.recipe

```diff
@@ -100,32 +100,33 @@ BUILD()
 INSTALL()
 {
 	cd build
+	mkdir -p $appsDir/Trojita
 	make install
 
 	#Moving things around
-	BINDIR="$appsDir/Trojita"
-	mv "$BINDIR/bin/be.contacts" "$BINDIR/be.contacts"
-	mv "$BINDIR/bin/trojita" "$BINDIR/Trojita"
-	strip "$BINDIR/be.contacts"
-	strip "$BINDIR/Trojita"
-	rm -rf "$BINDIR/bin"
+	
+	BINDIR=$appsDir/Trojita	
+	mv $BINDIR/bin/be.contacts $appsDir/Trojita
+	mv $BINDIR/bin/trojita $appsDir/Trojita/Trojita
+#	strip "$BINDIR/be.contacts"
+#	strip "$BINDIR/Trojita"
+	rm -rf $BINDIR/bin
 	rm -rf $dataDir/Trojita/{appdata,applications,icons}
 
 	# Add Haiku resources
-	local APP_SIGNATURE="application/x-vnd.flaska.org-Trojita"
-	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
-	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
-	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	local APP_SIGNATURE="application/x-vnd.focuswriter"
+	local MAJOR="0"
```

Comment:
Why are you hardcoding this? You actually deleted the automatisation and replaced with hardcoded things. I don't see the point.

Reply:
Because it can't parse "_git" part and fails. We need to improve this mechanism to work at all times.

Reply:
This doesn't answer however why local MIDDLE was removed...

Reply:
Actually it should be ~git.
But your point still true.